docs/docs/07-cli.md

```diff
@@ -28,17 +29,16 @@ ignite scaffold chain github.com/username/mars
 
 **SEE ALSO**
 
-* [ignite account](#ignite-account)	 - Commands for managing Ignite accounts
-* [ignite chain](#ignite-chain)	 - Build, initialize and start a blockchain node or perform other actions on the blockchain
-* [ignite completion](#ignite-completion)	 - Generate the autocompletion script for the specified shell
-* [ignite docs](#ignite-docs)	 - Show Ignite CLI docs
-* [ignite generate](#ignite-generate)	 - Generate clients, API docs from source code
-* [ignite node](#ignite-node)	 - Make calls to a live blockchain node
-* [ignite relayer](#ignite-relayer)	 - Connect blockchains by using IBC protocol
-* [ignite scaffold](#ignite-scaffold)	 - Scaffold a new blockchain, module, message, query, and more
-* [ignite tools](#ignite-tools)	 - Tools for advanced users
-* [ignite version](#ignite-version)	 - Print the current build information
-
+- [ignite account](#ignite-account) - Commands for managing Ignite accounts
+- [ignite chain](#ignite-chain) - Build, initialize and start a blockchain node or perform other actions on the blockchain
+- [ignite completion](#ignite-completion) - Generate the autocompletion script for the specified shell
+- [ignite docs](#ignite-docs) - Show Ignite CLI docs
+- [ignite generate](#ignite-generate) - Generate clients, API docs from source code
+- [ignite node](#ignite-node) - Make calls to a live blockchain node
+- [ignite relayer](#ignite-relayer) - Connect blockchains by using IBC protocol
+- [ignite scaffold](#ignite-scaffold) - Scaffold a new blockchain, module, message, query, and more
+- [ignite tools](#ignite-tools) - Tools for advanced users
+- [ignite version](#ignite-version) - Print the current build information
```

Comment:
We should add something for `ignite network`

Reply:
Doesn't it also make sense to add a section for `network` cli in this document as well?

Reply:
we don't offer a full spec of the network commands details right now
The purpose of the PR is a new network section which is a general guide for launching a chain, without the details on all command flags for example.

I think we should wait to make the command set for network more stable before working on such a spec